docs/01-getting-started.md

```diff
@@ -2,21 +2,23 @@
 
 ## Creating Your First Project
 
-The quickest way to get up and running with Studio is to clone an existing Git repository. If you're running from the Studio desktop version, then this can be as easy as choosing a location on your computer where a repository already exists. If you're running from the browser, this can be as simple as pasting the URL to any Github repository that you'd like to add. Give it a shot with a sample project we've provided below.
+The quickest way to get up and running with Studio is to clone an existing Git repository. If you're running Studio Desktop then this could be choosing a location on your computer where a repository already exists. If you're running Studio Web you could copy the URL to any GitHub, Bitbucket, GitLab, Gitea, or other Git repository. 
 
+Give it a shot with our sample project: 
+
+```
 https://github.com/stoplightio/studio-demo
+```
 
-Paste the URL above into the "Open Git Project" interface on the [Studio landing screen](https://stoplight.io/p/studio) and click "Clone".
+Paste the URL above into the "Open Git Project" form when you open Studio, and click "Clone".
 
 ![Start from an existing Git repo](../assets/images/studio-open-git-project-pre-filled.png)
 
-> Even if you decide to go the "New Local Project" route, if 'Include tutorial files?' is checked then you're actually cloning the files from the Stoplight [public Studio repository](https://github.com/stoplightio/studio) (which also happens to be where this documentation is hosted). Pretty neat, right?
-
 Assuming everything went to plan, you should now see a page resembling the one below.
 
 ![Overview of the Studio landing page](../assets/images/studio-web-landing.png)
 
-> Note that the screenshot above was taken from the Studio web interface, so there may be some slight differences if you are using the Studio desktop application.
+> Note that the screenshot above is of Studio Web, so there may be some slight differences if you are using Studio Desktop.
```

Comment:
This is a really old screenshot of Studio Web v1. Think you could update it to a Studio Desktop screenshot?

Reply:
We've got an issue for doing that somewhere, just tweaking some words in passing.